LoopMusic.xcodeproj/project.pbxproj

```diff
@@ -181,6 +182,9 @@
 		39EB89B5247A25D8002E0901 /* DoubleTextFieldOptionalSettingView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DoubleTextFieldOptionalSettingView.swift; sourceTree = "<group>"; };
 		39EC8AE1245A6E0F00D5240D /* PlaylistListViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = PlaylistListViewController.swift; sourceTree = "<group>"; };
 		39F1799922DBCD4800D3CDFF /* AudioEngine.c */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.c; path = AudioEngine.c; sourceTree = "<group>"; };
+		92561C0E24BF7EB500AA807E /* fa-solid-900.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; name = "fa-solid-900.ttf"; path = "../../../fa-glyphs/fa-solid-900.ttf"; sourceTree = "<group>"; };
```

Comment:
The font file is not included. It's a symlink from outside the project directory.